### PR TITLE
Dev/2.0 q work

### PR DESCRIPTION
**Pull request by alexjhawk**

### CODE REVIEW

**Comment by alexjhawk** [comment]
Hold off on the review for a couple of minutes. I thought I ran code formatting, but it appears I missed that.

**Comment by alexjhawk** [comment]
> Hold off on the review for a couple of minutes. I thought I ran code formatting, but it appears I missed that.

Corrected and now ready for review. Google's IDE plugin for google java format does not seem to catch all formatting issues now.

**Review by oliver-walker-hms — DISMISSED** [review]

**Review by TomKimsey — CHANGES_REQUESTED** [review]
Why is there no build check being run?

**Comment by alexjhawk** [comment]
> Why is there no build check being run?

Fixed - branch name was not correct in the action config

File: src/com/hms_networks/americas/sc/thingworx/data/TWApiManager.java [hunk 1]
```diff
@@ -46,6 +62,64 @@ public static String getApiDeviceName() {
...
+              // Copy pending payloads from data manager
+              List pendingPayloads = new ArrayList(TWDataManager.getPayloadsToSend());
+
+              // Iterate through each pending payload and send
+              Iterator pendingPayloadsIterator = pendingPayloads.iterator();
+              while (pendingPayloadsIterator.hasNext()) {
+                TWDataPayload dataPayload = (TWDataPayload) pendingPayloadsIterator.next();
+
+                // Send to Thingworx
+                boolean successful = sendJsonToThingworx(dataPayload.getPayloadString());
```
[the remark this answers is not in the document]
Comment (alexjhawk) [reply src/com/hms_networks/americas/sc/thingworx/data/TWApiManager.java]: changed

File: src/com/hms_networks/americas/sc/thingworx/TWApiManager.java [hunk 2]
```diff
@@ -73,15 +91,24 @@ private static synchronized void sendJsonToThingworx(String json) {
     }
 
     String response = null;
+    boolean successful = true;
```
[the remark this answers is not in the document]
Comment (alexjhawk) [reply src/com/hms_networks/americas/sc/thingworx/TWApiManager.java]: changed

**Review by oliver-walker-hms — APPROVED** [review]

**Review by TomKimsey — APPROVED** [review]

**Pull request merged**